db/seeds.rb

```diff
@@ -152,3 +198,12 @@
     phone: '012-345-6789', reservations: r_and_d.sample, delivery: r_and_d.sample
     )
 end
+
+user = User.new(email: 'seed@seeder.com', password: 'seedpassword')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.